userguide/virtualmachines.rst

```diff
@@ -61,7 +61,68 @@ and |ui-add| to open the wizard shown in
 .. figure:: images/virtual-machines-add-wizard-type.png
 
    Add VM
-
+   
+   
+Create a virtual machine by following these seven steps of the creation
+wizard:
+
+#. **Select VM wizard type**
```

Comment:
Rather than number options, why not just bullets for the steps?  In RST, those are just stars at the left margin.

Reply:
The wizard in the GUI has numbered steps. Thus, I tried to keep it related. Do you still want them to be bullets?

Reply:
I feel like bullets are less intimidating than a long list of numbers, but the numbers will let them keep track of where they are in the sequence.